FEM/rf_pcs.cpp

```diff
@@ -4364,10 +4364,16 @@ void CRFProcess::CheckExcavedElement()
 		if ((int)elem->GetPatchIndex() == ExcavMaterialGroup && elem->GetExcavState() == -1) // WX:04.2012
 		{
 			double const* ele_center(elem->GetGravityCenter());
-			if ((GetCurveValue(ExcavCurve, 0, aktuelle_zeit, &valid) + ExcavBeginCoordinate)
-			        > (ele_center[ExcavDirection])
-			    && (ele_center[ExcavDirection] - ExcavBeginCoordinate) > -0.001)
+			const double expected_coordinate =
```

Comment:
i see these lines several times. can you create a function for this and remove repeated codes?

Reply:
Created a new function.